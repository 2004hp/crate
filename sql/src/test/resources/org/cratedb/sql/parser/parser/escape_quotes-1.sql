select * from t1 where t1.a='A''s Escape Plan'