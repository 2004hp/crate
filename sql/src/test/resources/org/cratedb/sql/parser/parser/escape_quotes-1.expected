org.cratedb.sql.parser.parser.CursorNode@6f658567
name: null
updateMode: UNSPECIFIED
statementType: SELECT
resultSet: 	
	org.cratedb.sql.parser.parser.SelectNode@72196653
	isDistinct: false
	resultColumns: 		
		org.cratedb.sql.parser.parser.ResultColumnList@618d0daa

		[0]:		
		org.cratedb.sql.parser.parser.AllResultColumn@737c45ee
		tableName: null
		exposedName: null
		name: null
		tableName: null
		isDefaultColumn: false
		type: null
	fromList: 		
		org.cratedb.sql.parser.parser.FromList@63d3477d

		[0]:		
		org.cratedb.sql.parser.parser.FromBaseTable@43fa92be
		tableName: t1
		updateOrDelete: null
		null
		correlation Name: null
		null
	whereClause: 		
		org.cratedb.sql.parser.parser.BinaryRelationalOperatorNode@77b81acf
		operator: =
		methodName: equals
		type: null
		leftOperand: 			
			org.cratedb.sql.parser.parser.ColumnReference@57e84970
			columnName: a
			tableName: t1
			type: null
		rightOperand: 			
			org.cratedb.sql.parser.parser.CharConstantNode@532d8526
			value: A's Escape Plan
			type: CHAR(15) NOT NULL
